elixir/apps/web/test/web/live/actors/show_test.exs

```diff
@@ -524,7 +543,14 @@ defmodule Web.Live.Actors.ShowTest do
              |> element("#identity-#{google_identity.id} button", "Send Welcome Email")
              |> has_element?()
 
-      refute lv
+      assert lv
+             |> element(
+               "#identity-#{oidc_manually_provisioned_identity.id} button",
+               "Send Welcome Email"
+             )
+             |> has_element?()
+
+      assert lv
```

Comment:
Are there any cases now where the "Send Welcome Email" button will not be shown?  If so, do we need to add a test case?

Reply:
Yeah it looks like it. Let me add that real quick.